source/core/src/main/com/csse3200/game/components/screendisplay/LoseScreenDisplay.java

```diff
@@ -68,21 +98,47 @@ private void addActors() {
         animals.add("Snake");
         animals.add("Dino");
         animals.add("Minotaur");
+        animals.add("Dragon");
+        animals.add("Werewolf");
+        animals.add("Cthulu");
+        animals.add("Kitsune");
+        animals.add("Birdman");
+
         animalDeathScreens.add("You thought you were the predator, but turns out you are just prey!");
         animalDeathScreens.add("Too slow, too soft. You didn't stand a chance against my claws!");
         animalDeathScreens.add("Couldn't see in the dark, huh? My world, my rules. Better luck next time, human!");
         animalDeathScreens.add("Fetch? More like fight! Too bad you didn't put up much of one.");
         animalDeathScreens.add("Slither, slither... you never saw me coming, and now it is lights out for you!");
         animalDeathScreens.add("Extinct? Not me! Guess it is you who is heading for the history books!");
         animalDeathScreens.add("Lost in my labyrinth of power, and now you are just another trophy!");
-        table = new Table();
-        table.setFillParent(true);
-        LastAttackAnimal = readClass(String.class, "configs/LastAttack.json", FileLoader.Location.EXTERNAL);
+        animalDeathScreens.add("Foolish mortal, my fire has forged empires and destroyed heroes. What made you think you could survive?");
+        animalDeathScreens.add("Ah, the moon's glow reveals the truth: you were never a match for the beast within me!");
+        animalDeathScreens.add("You are but a fleeting whisper in the vast eternity of my dreams. Insignificant, erased!");
+        animalDeathScreens.add("Deceived by shadows and tricked by tales, your end came at the hands of my cunning!");
+        animalDeathScreens.add("From the skies I watched, and with swift wings I struck. Did you really think you could escape my gaze?");
+
+
+
+        LastAttackAnimal = readClass(String.class, "configs/LastAttack.json", FileLoader.Location.LOCAL);
+        if(!LastAttackAnimal.equals("Unknown")){
+            int firstSpaceIndex = LastAttackAnimal.indexOf(' ');
+            if (firstSpaceIndex != -1) { // Check if there is at least one space in the string
+                LastAttackAnimal = LastAttackAnimal.substring(0, firstSpaceIndex);
+            }
+            LastAttackAnimal = capitalizeFirstChar(LastAttackAnimal);
+            System.out.println(LastAttackAnimal);
```

Comment:
use a log message instead of a print, that gives you the option to turn it off if you want to